Doc/library/imaplib.rst

```diff
@@ -353,16 +364,21 @@ An :class:`IMAP4` instance has the following methods:
    Send ``NOOP`` to server.
 
 
-.. method:: IMAP4.open(host, port)
+.. method:: IMAP4.open(host, port, timeout=None)
 
    Opens socket to *port* at *host*.  This method is implicitly called by
    the :class:`IMAP4` constructor.  The connection objects established by this
    method will be used in the :meth:`IMAP4.read`, :meth:`IMAP4.readline`,
-   :meth:`IMAP4.send`, and :meth:`IMAP4.shutdown` methods.  You may override
-   this method.
+   :meth:`IMAP4.send`, and :meth:`IMAP4.shutdown` methods.
+   The optional *timeout* parameter specifies a timeout in seconds
+   for the connection attempt. If timeout is not given or is None,
+   the global default socket timeout is used.
+   You may override this method.
```

Comment:
It seems like this line was supposed to follow the previous text (`:meth:`IMAP4.shutdown` methods.) Looks out of place here.